homeassistant/components/amberelectric/__init__.py

```diff
@@ -0,0 +1,50 @@
+"""Support for Amber Electric."""
+
+import logging
+
+from amberelectric import ApiException, Configuration
+from amberelectric.api import amber_api
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import CONF_API_TOKEN, CONF_SITE_ID, DOMAIN
+
+PLATFORMS = ["sensor"]
+LOGGER = logging.getLogger(__package__)
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Amber Electric from a config entry."""
+    configuration = Configuration(access_token=entry.data.get(CONF_API_TOKEN))
```

Comment:
Use square brackets if you know the key exists. 
```suggestion
    configuration = Configuration(access_token=entry.data[CONF_API_TOKEN)
```